mm/pmm.c

```diff
@@ -570,3 +570,9 @@ void map_used_memory(void) {
         }
     }
 }
+
+void map_frames_array(void) {
+    frames_array_t *array;
+
+    list_for_each_entry (array, &frames, list) { kmap_4k(virt_to_mfn(array), L1_PROT); }
```

Comment:
Nit: wrap lines, like:
```c
list_for_each_entry (array, &frames, list) {
    kmap_4k(virt_to_mfn(array), L1_PROT);
}
````

Could even spare the brackets, but I guess that's part of the coding style in KTF?

Reply:
Yes, I know... but it is the bloody `.clang-format` that enforces this and I cannot easily teach it to do the right thing. It annoys me too, but so far I have been too lazy to look into details of the configuration.

Reply:
:see_no_evil: